kafka/producer/base.py

```diff
@@ -168,12 +179,10 @@ def send_messages(self, topic, partition, *msg):
         @param: *msg, one or more message payloads -- type bytes
         @returns: ResponseRequest returned by server
         raises on error
-
```

Comment:
Nit: Let's reduce the white space changes as well. It makes future merging harder.